bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -56,38 +56,44 @@ class AppContainer extends Component {
 
 export default withRouter(injectIntl(withModalMounter(createContainer((
   { router, intl, mountModal, baseControls }) => {
-    // Check if user is kicked out of the session
+
+  // Displayed error messages according to the mode (kicked, end meeting)
+  let clearCredential = (mode) => {
+    Auth.clearCredentials()
+        .then(() => {
+          router.push('/error/403');
+          baseControls.updateErrorState(
+            mode == 'kicked' ?
```

Comment:
You probably should pass the message instead of a `mode` is more flexible. And 403 dont really makes sense for when the meeting ends, 410 looks more suitable.

```
const sendToError = (code, message) => {
  Auth.clearCredentials()
    .then(() => {
      router.push(`/error/${code}`);
      baseControls.updateErrorState(message);
    });
};
...
sendToError(403, intl.formatMessage(intlMessages.kickedMessage));
...
sendToError(410, intl.formatMessage(intlMessages.endMeetingMessage));
```